EncounterMe/EncounterMe/Views/RoutesPage.xaml.cs

```diff
@@ -38,7 +38,7 @@ IEnumerable<MapPin> GetAllObjects(string searchText = null)
             }
             _textValue = searchText;
             var objectsQuery = from mapPin in _myPinList.ListOfPins
-                               where mapPin.Address.City.ToLower().Contains(searchText.ToLower())
+                               where mapPin.Address.City != null && mapPin.Address.City.ToLower().Contains(searchText.ToLower())
```

Comment:
Šiaip teoriškai neturėtų būt tokių objektų, kurie neturi miesto, nes pridedant objektus mes gi pachekinam ar pilnas adresas. Tai įtariu, kad testiniais sumetimais pridėti objektai yra nesamonė, tai all good :DD

Reply:
Čia įdėjau šitą, nes kol nebuvo tikrinimo normalaus, aš pridėjau tų objektų, kurie neturi miesto, ir dabar išmeta exception jei bandai ieškot :D